lib/Display/display.cpp

```diff
@@ -5,11 +5,14 @@ Adafruit_ILI9341 tft = Adafruit_ILI9341(10, 9, -1);
 
 display::display() {}
 
+
 void display::drawDisplay(int returnInformation[], Item items[], int sizeOfArray, long ms, long s)
 {
     milliSeconds = ms;
     seconds = s;
 
+    time();
```

Comment:
call this print time or something similar.

Reply:
I did it the same way as the score function to keep parity, but changing it would be fine by me aswell